src/dataframe/dataframe.jl

```diff
@@ -1115,7 +1119,88 @@ function Base.push!(df::DataFrame, row::Union{AbstractDict, NamedTuple})
     df
 end
 
-# array and tuple like collections
+"""
+    push!(df1::DataFrame, iterable::Any)
+    push!(df::DataFrame, row::Union{AbstractDict, NamedTuple, DataFrameRow}; exact::Bool=true)
+
+Add in-place one row at the end of `df1` taking the values from the iterable `iterable`
+or `row`.
+
+Adding values from `iterable` is based on column number.
```

Comment:
Instead of using two different names to refer to the same argument, I'd say "if `row` is a ...; else ...".

Reply:
Fixed